Header header
int32 distHorz
#distance in mm
int32 pdoaHorz
#phase difference of arrival in mDeg
int32 distVert
#distance in mm
int32 pdoaVert
#phase difference of arrival in mDeg
geometry_msgs/Pose pose
